charts/rs-server-station-secrets/values.yaml

```diff
@@ -181,4 +184,94 @@ app:
         password: test
         client_id: client_id
         client_secret: client_secret
-        authorization: Basic test
\ No newline at end of file
+        authorization: Basic test
+    cadip_session:
```

Comment:
@agrosu1978 @Padeanu why are the cadip "_session" stations defined in rs-demo/local-mode/config/rs-server.yaml but were missing from the helm charts ? Do we need them ? 

See the comment here: https://github.com/RS-PYTHON/rs-server/blob/develop/services/common/rs_server_common/authentication/authentication_to_external.py#L452
```python
# TODO: This part of the code has to be deleted after the cadip_ws_config(_token_module) files
# will be modified. The parameters from the cadip_session sections (ins, mti...) should replace those params
# from cadip sections (ins, mti...)
```
So what should we do with these ?

Reply:
_sessions are defined in helm chart.